src/main/resources/anniv6Resources/localization/eng/BrokenSpace/Relicstrings.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
I'm going to give some thoughts on how I see the balance and play experience for these relics.

One thing to keep in mind with this is that if there's a relic people almost never want to pick up, or that leads to a negative experience for a lot of players, it's probably best to leave it out of the mod. Relics having major upsides and downsides is perfectly fine, but when things tip over into downsides being so large that the upside almost never justifies it or it feels bad, that's no good. Going too far in the other direction (i.e. very overpowered relics) is also bad).

That gets to the heart of my overall feedback for these relics: most of them go too far on the upsides and/or downsides, and would be better with less extreme effects (which can still be really impactful). In a number of cases this just means number tweaks or slight changes, but some relics might need a redesign. I do think there are a good number of cool ideas here, and this feedback is intended to help figure out how to execute the cool ideas in a way that players will appreciate, alongside identifying ideas that are better off being reworked so that people will appreciate what your zone does.

So with all that said, here are my thoughts, relic by relic:
* Broken Data Disk: This feels like it will too often be really bad or really good. -3 focus is enough that by default, Lightning and Frost Orbs will do nothing for their passive. But if you get back to positive focus with powers or clearing the debuff, you can quickly become overpowered. It's a bit too swingy. Maybe something like giving -1 focus in return for doubling the first orb channeled each turn would be sensible?

* Broken Snecko Skull: This is probably fine. There's some broken combinations with this (like Intangible), but the average power level seems okay.

* Broken Red Skull: This is probably fine. It might be too good at letting you stay ahead when you're already in good shape, but then again dexterity isn't that good for Ironclad. Easy enough to adjust later if needed.

* Broken Omamori: It looks like the description of this relic says it upgrades 1 card per rest, but the code upgrades 2 cards per rest. I'm not sure which is intended, but 3 curses is a lot, so I don't think I'd ever take this if it only upgraded 1 card. At 2 cards upgraded per rest, maybe... but 3 curses is such a short-term disadvantage that I still might not. Toning down the numbers might make this better.

* Broken Damaru: This seems insanely strong. Starting combat in Divinity will trivialize hallway fights and even some elites! Getting this relic is like triple Lantern just for the energy gain, and the 3x damage doesn't even have another relic you can compare it to. A different design might be best.

* Broken Wing Boots: The player won't have a clear idea of what this does and I'm not sure it's worth taking, which means this probably isn't a good idea. Best to find a different design.

* Broken Sozu: This effect is weird enough that I'm not at all sure about it. It fits the pattern of being too swingy, though; if you get this close to the end of the game, there's almost no downside. I suspect this would be better off with a less extreme design.

* Broken Snecko Eye: Setting aside whether this relic is good or bad, I'm concerned that it isn't fun to play with. Drawing cards that are almost completely disguised means either guessing blindly or counting cards to work out what they are (by looking at your hand/draw/discard to eliminate known cards until all that's left is the unreadable ones). Guessing blindly feels bad, counting cards is a play pattern we don't want to encourage. I'm also concerned about the unreadable card effect; having the card text constantly changing is no good. This one might need a rework.

* Broken Maw Bank: Seems reasonable. 500 gold might be a bit too much compared to Old Coin; the downside of needing to spend it all won't be a big deal a lot of the time (just buying out relics and potions and removal can easily be 800+ gold happily spent). Maybe try this at 400 gold, it's easy to tweak if it feels too weak.

* Broken Prayer Wheel: The downside here is excessive. Getting no card rewards is such a large drawback that taking this will doom most runs. Even with the 4x gold, unless you have Courier you're not likely to be able to buy enough cards from shops to make a deck that can win at high ascension (there's a reason Busted Crown is one of the worst boss relics). I think this would be a more interesting and balanced relic if it reduced the number of cards in rewards by 1, and had a lower multiplier for gold (e.g. 2).

* Broken Matryoshka: This relic has two issues. First, while the idea is reasonable, 5 relics for 2 empty chests is too good. Second, this relic is a forced skip if it drops after the act 1 chest for anyone planning to fight the Heart (since there's only two guaranteed chests left at that point, and if both are empty you have no way to get the blue key). Not everyone goes for the Heart fight, but a relic that a substantial subset of players almost always skips is no good. However, this can be fixed, as follows.
  * For the first problem, something like 3 relics for 2 empty chests seems balanced (immediate benefit now, neutral on relics in the long run, miss out on the gold that might be in the chests). Alternately, you could make it 4 relics, but always 3 commons and 1 uncommon (which matches Matryoshka giving a 75% chance of common and 25% chance of uncommon).
  * For the second problem, you would need to have one of the relics show up linked with the Blue Key (and say that in the relic description so that people going for the Heart understand they can take this relic).

* Broken Sundial: This seems like a relic you would almost always skip. Shuffling your discard pile back into your draw pile constantly puts you at risk of never finding your key cards in elite and boss fights. I recognize that like some of the other relics here, there's very specific synergies that could make this work, but that's not enough if the normal reaction is closer to "that's bad, skip". Might be best off finding a new design.

Reply:
Relic balance seems reasonable after the changes (though Broken Wing Boots may need more discussion for other reasons)

Thanks for going through and responding to all the feedback!